cmake/SundialsSetupCompilers.cmake

```diff
@@ -476,20 +476,42 @@ endforeach()
 # Configure clang-tidy for linting
 # ===============================================================
 
+set(SUNDIALS_DEV_CLANG_TIDY_DIR ${CMAKE_BINARY_DIR}/clang-tidy/)
+
 if(SUNDIALS_DEV_CLANG_TIDY)
-  make_directory(${CMAKE_BINARY_DIR}/clang-tidy)
+  find_program(CLANG_TIDY_PATH NAMES clang-tidy)
+  if(NOT CLANG_TIDY_PATH)
+      message(FATAL_ERROR "Could not find the program clang-tidy")
+  endif()
+  message("Found IWYU: ${CLANG_TIDY_PATH}")
+  make_directory(${SUNDIALS_DEV_IWYU_DIR})
+  make_directory(${SUNDIALS_DEV_CLANG_TIDY_DIR})
   if(SUNDIALS_DEV_CLANG_TIDY_FIX_ERRORS)
-    set(CMAKE_C_CLANG_TIDY clang-tidy -format-style='file' --fix)
-    set(CMAKE_CXX_CLANG_TIDY clang-tidy -format-style='file' --fix)
+    set(CMAKE_C_CLANG_TIDY ${CLANG_TIDY_PATH} -format-style='file' --fix)
+    set(CMAKE_CXX_CLANG_TIDY ${CLANG_TIDY_PATH} -format-style='file' --fix)
   else()
-    set(CMAKE_C_CLANG_TIDY clang-tidy 
-      -format-style='file' 
-      --export-fixes=${CMAKE_BINARY_DIR}/clang-tidy/clang-tidy-fixes.yaml
+    set(CMAKE_C_CLANG_TIDY ${CLANG_TIDY_PATH}
+      -format-style='file'
+      --export-fixes=${SUNDIALS_DEV_CLANG_TIDY_DIR}/clang-tidy-fixes.yaml
     )
-    set(CMAKE_CXX_CLANG_TIDY 
-      clang-tidy 
-      -format-style='file' 
-      --export-fixes=${CMAKE_B_DIR}/clang-tidy/clang-tidy-cxx-fixes.yaml
+    set(CMAKE_CXX_CLANG_TIDY
+      ${CLANG_TIDY_PATH}
+      -format-style='file'
+      --export-fixes=${SUNDIALS_DEV_CLANG_TIDY_DIR}/clang-tidy-cxx-fixes.yaml
     )
   endif()
 endif()
+
+if(SUNDIALS_DEV_IWYU)
```

Comment:
```suggestion
set(SUNDIALS_DEV_IWYU_DIR ${CMAKE_BINARY_DIR}/iwyu/)

if(SUNDIALS_DEV_IWYU)
```

Reply:
Actually, is `SUNDIALS_DEV_IWYU_DIR` needed? IWYU doesn't seem to have an option to write its suggestions to a file.

Reply:
As you have noted, it is not needed.